stdlib/private/SwiftPrivatePthreadExtras/SwiftPrivatePthreadExtras.swift

```diff
@@ -85,7 +85,7 @@ internal func _make_pthread_t() -> pthread_t {
 #if os(Linux)
   return pthread_t()
 #else
-  return nil
+  return 0
```

Comment:
This is not correct. Android should be in the Linux path here. Darwin and FreeBSD's `pthread_t`s are pointers.